src/Components/Header/Header.css

```diff
@@ -0,0 +1,36 @@
+.vote-image {
+  width: 80px;
+  height: 80px;
+}
```

Comment:
I am absolutely terrible with CSS, but my mentor showed me a cool trick around using 'rem' for much of my app styling.  Essentially you declare  a 'default' pixel size within app.css and then 1 rem is equal to that value.  That way if you want to change the scale of your application (most commonly by making the site responsive), you are working with a percentage instead of a hard coded pixel size.

Reply:
If you'd like to walk me through this once I've added the full address fields to the form, I'd really appreciate it!